src/main/java/org/jenkinsci/maven/plugins/hpi/hosting/NoDashPluginOrJenkins.java

```diff
@@ -0,0 +1,16 @@
+package org.jenkinsci.maven.plugins.hpi.hosting;
+
+import org.apache.maven.project.MavenProject;
+import org.jenkinsci.maven.plugins.hpi.hosting.VerificationMessage.Severity;
+
+public class NoDashPluginOrJenkins implements VerificationRule {
+
+	@Override
+	public VerificationMessage validate(MavenProject project) {
+		if (project.getArtifactId().contains("plugin") || project.getArtifactId().contains("jenkins")) {
```

Comment:
modified to use `-plugin` and `-jenkins`. There would still be warnings, but in (very) rare cases for the existing plugins and the potentially future ones.